.github/workflows/verify-ec2.yml

```diff
@@ -0,0 +1,71 @@
+---
+name: CIS AWS Foundations v2.0.0
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  my-job:
+    name: Validate my profile
+    runs-on: ubuntu-latest
+    env:
+      CHEF_LICENSE: accept-silent
+      CHEF_LICENSE_KEY: ${{ secrets.SAF_CHEF_LICENSE_KEY }}
+      RESULTS_FILE: ${{ github.sha }}-inspec_results.json
+      PROFILE_FILE: profile.json
+      INPUT_FILE: testing.inputs.yml
+      HEIMDALL_URL: https://heimdall-demo.mitre.org
+    steps:
+      - name: add needed packages
+        run: sudo apt-get install -y jq
+      - name: Configure AWS credentials
```

Comment:
The logs seems to indicate this may be unneeded given they seem to be pre installed on the runner